src/led.c

```diff
@@ -47,24 +47,25 @@ static const uint32_t led_port[] = {
 static const struct led_table {
 	uint8_t port_idx;
 	uint8_t pin;
+	enum bl_acq_source pd;
 } led_table[BL_LED_COUNT] = {
 #if (BL_REVISION == 1)
-	[ 0] = { .port_idx = LED_PORT_B, .pin =  1 },
-	[ 1] = { .port_idx = LED_PORT_B, .pin =  2 },
-	[ 2] = { .port_idx = LED_PORT_B, .pin = 10 },
-	[ 3] = { .port_idx = LED_PORT_B, .pin = 11 },
-	[ 4] = { .port_idx = LED_PORT_A, .pin =  8 },
-	[ 5] = { .port_idx = LED_PORT_A, .pin =  9 },
-	[ 6] = { .port_idx = LED_PORT_A, .pin = 10 },
-	[ 7] = { .port_idx = LED_PORT_A, .pin = 15 },
-	[ 8] = { .port_idx = LED_PORT_C, .pin = 13 },
-	[ 9] = { .port_idx = LED_PORT_C, .pin = 14 },
-	[10] = { .port_idx = LED_PORT_C, .pin = 15 },
-	[11] = { .port_idx = LED_PORT_B, .pin =  9 },
-	[12] = { .port_idx = LED_PORT_B, .pin =  8 },
-	[13] = { .port_idx = LED_PORT_B, .pin =  7 },
-	[14] = { .port_idx = LED_PORT_B, .pin =  6 },
-	[15] = { .port_idx = LED_PORT_B, .pin =  5 },
+	[ 0] = { .port_idx = LED_PORT_B, .pin =  1, .pd = BL_ACQ_PD3 },
+	[ 1] = { .port_idx = LED_PORT_B, .pin =  2, .pd = BL_ACQ_PD3 },
+	[ 2] = { .port_idx = LED_PORT_B, .pin = 10, .pd = BL_ACQ_PD3 },
+	[ 3] = { .port_idx = LED_PORT_B, .pin = 11, .pd = BL_ACQ_PD3 },
+	[ 4] = { .port_idx = LED_PORT_A, .pin =  8, .pd = BL_ACQ_PD4 },
+	[ 5] = { .port_idx = LED_PORT_A, .pin =  9, .pd = BL_ACQ_PD4 },
+	[ 6] = { .port_idx = LED_PORT_A, .pin = 10, .pd = BL_ACQ_PD4 },
+	[ 7] = { .port_idx = LED_PORT_A, .pin = 15, .pd = BL_ACQ_PD4 },
+	[ 8] = { .port_idx = LED_PORT_C, .pin = 13, .pd = BL_ACQ_PD2 },
+	[ 9] = { .port_idx = LED_PORT_C, .pin = 14, .pd = BL_ACQ_PD2 },
+	[10] = { .port_idx = LED_PORT_C, .pin = 15, .pd = BL_ACQ_PD2 },
+	[11] = { .port_idx = LED_PORT_B, .pin =  9, .pd = BL_ACQ_PD2 },
+	[12] = { .port_idx = LED_PORT_B, .pin =  8, .pd = BL_ACQ_PD1 },
+	[13] = { .port_idx = LED_PORT_B, .pin =  7, .pd = BL_ACQ_PD1 },
+	[14] = { .port_idx = LED_PORT_B, .pin =  6, .pd = BL_ACQ_PD1 },
+	[15] = { .port_idx = LED_PORT_B, .pin =  5, .pd = BL_ACQ_PD1 },
 #else
 	[ 0] = { .port_idx = LED_PORT_C, .pin = 14 },
```

Comment:
Please add the same affinities for revision 2.

Reply:
I have a branch with rev2 changes in, and I could squash those commits in